scripts/prepare.sh

```diff
@@ -2,4 +2,4 @@
 
 set -euo pipefail
 
-if git status &>/dev/null; then git config core.hooksPath .githooks; fi
+npx husky
```

Comment:
What if we remove this file and run `husky` in the `prepare` command instead?

Reply:
that works for me